abstract/trace-context-overview.md

```diff
@@ -1,7 +1,11 @@
 # Overview
 
-Trace context represented by set of name/value pairs describing identity of every http request. As a performance optimization measure few of these pairs were promoted to a separate header. This header has fixed length and defined sequence of fields.
+Trace context represented by set of name/value properties describing identity of every http request. Few of these properties are promoted to a separate header. It also allows to align different tracing frameworks on semantic of these properties. This is also done as a performance optimization measure - header has fixed length and defined sequence of fields. It is easy to parse and mutate this header.

```

Comment:
Stab at a different overview:

The trace context identifies an http request and its position in a potentially distributed operation. Its properties exist in a fixed-length generic header, `Trace-Context`, and possibly extend into a vendor-specific `Trace-Context-Ext` header. The `Trace-Context` defines a sequence of fields of fixed length all vendors must process, such as a trace ID. If a vendor cannot process `Trace-Context-Ext`, it must restart the trace. The following elaborates format details and propagation semantics implied by this standard.